assignments/assignments_parser_test.go

```diff
@@ -216,6 +221,93 @@ func TestParseUnknownFields(t *testing.T) {
 	}
 }
 
+func TestParseAndSaveAssignment(t *testing.T) {
```

Comment:
This is an excellent test for the expected case. I think we are still missing the bad case, similar to the example you explained in #999. 

Right now, I think `readTestsRepositoryContent` will return an error if a single `yaml` file has a mistake we don't tolerate, e.g., using `assignmentid` instead of `order`. Thus, I don't think we will be able to test the `db.UpdateAssignments` with one or more faulty assignments. That said, such a test should probably be done in the `database` package if we actually need it and perhaps in a separate PR. What's your take?